HarmonyHub/Entities/Activity.cs

```diff
@@ -19,7 +20,7 @@ public class Activity : IComparable<Activity>
         [DataMember(Name = "activityTypeDisplayName")]
         public string ActivityTypeDisplayName { get; set; }
         [DataMember(Name = "controlGroup")]
-        public IList<object> ControlGroup { get; set; }
+        public IList<object> ControlGroups { get; set; }
```

Comment:
You added ControlGroup as a type, I guess it can be used here instead of object too?